src/components/Button.js

```diff
@@ -9,14 +9,16 @@ import { OrientationContext } from '../OrientationProvider';
 
 export const Button = ({ title, onPress }) => {
   const { orientation } = useContext(OrientationContext);
+
   return (
     <RNEButton
       onPress={onPress}
       title={title}
-      titleStyle={styles.titleStyle}
-      containerStyle={
-        orientation === 'landscape' ? styles.containerStyleLandscape : styles.containerStyle
-      }
+      titleStyle={[styles.titleStyle, orientation === 'landscape' && styles.titleStyleLandscape]}
```

Comment:
here for title i used the same array stuff for applying styles

Reply:
thanks